src/components/search-drivers/context/search-drivers-provider.tsx

```diff
@@ -18,7 +20,17 @@ export const SearchDriversProvider: FC<Props> = ({ children }) => {
   const shouldRedirectOnSubmit = pathname !== PATHS.SELECT_DRIVER;
   const searchDriversValues = useSearchDriversFormik(shouldRedirectOnSubmit);
   const { i18n, t } = useTranslation();
-  const menuItems: SearchDestination[] = i18n.language === LANGUAGES.RU ? RUdestinations : ENdestinations;
+
+  const destinationsLocalization = i18n.language === LANGUAGES.RU ? "ru_destinations" : "en_destinations";
+  const [destinationsCollection, destinationsLoading] = useCollection(destinationsLocalization);
+
+  const destinations = collectDataFromCollection(destinationsCollection).map((destination: any) => {
```

Comment:
Никаких any. У нас есть специальный интерфейс для направлений, который надо доработать. У нас он был написан до изменения концепции
```suggestion
  const destinations = collectDataFromCollection(destinationsCollection).map((destination: SearchDestination) => {
```